runners/maven-plugin/src/main/kotlin/DokkaMojo.kt

```diff
@@ -152,6 +155,12 @@ abstract class AbstractDokkaMojo(private val defaultDokkaPlugins: List<Dependenc
     @Parameter
     var includeNonPublic: Boolean = DokkaDefaults.includeNonPublic
 
+    @Parameter(property = "visibility")
+    var documentedVisibilities: Set<DokkaConfiguration.Visibility> = DokkaDefaults.documentedVisibilities
+        // hack to set the default value for lists, didn't find any other safe way
```

Comment:
Why you do not use `defaultValue` in Parameter?

Reply:
I want to take the default value from `DokkaDefaults.documentedVisibilities` and you can't use it in annotations since it's a list and can't be a `const`. `defaultValue` can do some expression evaluation, but I didn't find a way to reference a field. The only thing I could write in there is `PUBLIC`, which works, but if defaults change / refactoring occurs - we could have a problem here

Reply:
In this case you need to manually set the default value in the POM.

Reply:
although it can be inconsistent with `DokkaDefaults.documentedVisibilities`